### DISCUSSION TITLE
Allow sorting leaderboards by metrics other than score

### DISCUSSION THREAD

**Discussion by trustytrojan** [ppy/osu#26110] [Ideas] [answered]
Mania player here. After the 2023.12.18 release where the DT/HT score multipliers were rolled back to how they behave in stable, I no longer find any motivation to play charts faster, which is making my experience worse.

What would bring back this motivation is allowing leaderboards to be sorted by metrics other than score: in my case, by pp, but other players may want to sort by accuracy *(which I suppose is useless considering score is 99% based on accuracy; however other rulesets can benefit!)*. It could be even more fine-grained than just those two: by number of Perfects hit, by Greats hit all the way down to misses (obviously sorted ascending), combo, etc.

It's also kind of inconvenient to have to click on every leaderboard entry to view the other stats of a score. Adding the option to sort by pp and display pp on leaderboard entries **as the main metric (replacing score)** would make my and maybe a few other players' experience better.

Some people might say "just buy supporter and filter by DT/HT", but that's still a score-sorted leaderboard, and I'm proposing this as a free-to-use feature. Heck, I'll consider buying supporter if this is implemented as a supporter-only feature.

**Comment by peppy** [comment, marked as answer]
This is planned eventually.

**Reply by trustytrojan** [reply]
Thank you lord peppy.
